docs/en/pimoroni_trackball.md

```diff
@@ -31,12 +31,12 @@ trackball = Trackball(
     # optional: set rotation angle of the trackball breakout board, default is 1
     angle_offset=1.6, 
     handlers=[
-        # act like an encoder, input arrow keys
+        # act like an encoder, input arrow keys, enter when pressed
         KeyHandler(KC.UP, KC.RIGHT, KC.DOWN, KC.LEFT, KC.ENTER), 
-        # on layer 1 and above use the default pointing behavior
-        PointingHandler(),
-        # use ScrollDirection.NATURAL (default) or REVERSE to change the scrolling direction
-        ScrollHandler(scroll_direction=ScrollDirection.NATURAL)
+        # on layer 1 and above use the default pointing behavior, left click when pressed
+        PointingHandler(press=KC.MB_LMB),
+        # use ScrollDirection.NATURAL (default) or REVERSE to change the scrolling direction, left click when pressed
+        ScrollHandler(scroll_direction=ScrollDirection.NATURAL, press=KC.MB_LMB)
```

Comment:
This needs better documentation. It looks like the `press` argument is required, and since no one is looking up the source code when using KMK for the first time, they will copy-paste anything without a care in the world.
Maybe a small subsection about existing handlers, their arguments and an example of a custom one? (That would also reduce the need for a customizable press event)